README.md

```diff
@@ -38,32 +41,40 @@ OTLP is defined in terms of Google protocol buffer definitions.
 
 OTLP is a stateless protocol, where export requests map directly into
 the data model, nothing is omitted, and little is shared.  OTLP export
-requests to not contain external or internal references, making the
+requests do not contain external or internal references, making the
 data relatively simple and easy to interpret.  Because of this design,
 users of OTLP will typically configure network compression.  In
 environments where telemetry data will be shipped to a service
 provider across a wide-area network, users would like more compression
 than can be achieved using a row-based data model and a stateless protocol.
 
-## Project goals
+## Project phases
+
+### Phase 1
 
-This is organized in phases. Our initial aim is to facilitate traffic reduction
-between a pair of OpenTelemetry collectors as illustrated in the following
+Phase 1 of the project is complete. Our initial aim was to facilitate traffic reduction
```

Comment:
Nit:
In section "Project Status", the following wordings still exist. (line 111)
_The first phase of the project has entered the [Beta stability level,_

Would be good to modify them accordingly.

Reply:
I've updated the phrasing to clearly state that phase-1 deliverables are in Beta status.